tests/Fixtures/index.md

```diff
@@ -2,17 +2,19 @@
 title: API Reference
 
 language_tabs:
-- bash
-- javascript
+    - bash
```

Comment:
Any reason for these indentation changes? They're causing the build to fail.

Reply:
i think That was Prettier Auto formatting 
i'll fix them ASAP